MotMystere.cpp

```diff
@@ -2,19 +2,53 @@
 //
 
 #include <iostream>
+#include <string>
+#include <ctime>
+#include <cstdlib>
+
+using namespace std;
 
 int main()
 {
-    std::cout << "Hello World!\n";
+   
+    
+    //1 : On demande de saisir un mot
+
+    string motMystere("Procra");
```

Comment:
You never use the initial "Procra", so there is no need to initialize your string with "Procra"

Reply:
oks=)